src/System.Net.Security/src/System/Net/NegotiateStreamPal.Unix.cs

```diff
@@ -270,7 +270,7 @@ internal static Exception CreateExceptionFromError(SecurityStatusPal statusCode)
                 outputBuffer.size = outputBuffer.token.Length;
                 outputBuffer.offset = 0;
 
-                outFlags = ContextFlagsAdapterPal.GetContextFlagsPalFromInterop((Interop.NetSecurityNative.GssFlags)outputFlags);
+                outFlags = ContextFlagsAdapterPal.GetContextFlagsPalFromInterop((Interop.NetSecurityNative.GssFlags)outputFlags, isServer:false);
```

Comment:
Yes.. This is client side implementation so this will  always be false. When we will enable the AuthenticateAsServer for unix, it will be called as isserver true in that implementation.